testing/testing-utils/src/generators.rs

```diff
@@ -261,7 +266,7 @@ pub fn random_block_range<R: Rng>(
     rng: &mut R,
     block_numbers: RangeInclusive<BlockNumber>,
     block_range_params: BlockRangeParams,
-) -> Vec<SealedBlock> {
+) -> Vec<SealedBlock<Block>> {
```

Comment:
It might make sense to type alias SealedBlock in the future like Block and BlockBody